<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuonFitDataSelector</class>
 <widget class="QWidget" name="MuonFitDataSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MuonFitDataSelector</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutAll">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxDataSelector">
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutData">
        <item>
         <widget class="QGroupBox" name="groupBoxWorkspaces">
          <property name="title">
           <string>Workspaces</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayoutWorkspaces">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayoutRuns">
             <item>
              <widget class="QLabel" name="lblRuns">
               <property name="text">
                <string>Runs:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="MantidQt::MantidWidgets::MWRunFiles" name="runs" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="label" stdset="0">
                <string/>
               </property>
               <property name="multipleFiles" stdset="0">
                <bool>true</bool>
               </property>
               <property name="findRunFiles" stdset="0">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayoutRadioButtons">
             <item>
              <widget class="QRadioButton" name="rbCoAdd">
               <property name="text">
                <string>Co-add</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbSimultaneous">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Simultaneous</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayoutTime">
             <item>
              <widget class="QLabel" name="lblStart">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start (us):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtStart">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacerTime">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="lblEnd">
               <property name="text">
                <string>End (us):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtEnd">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxGroups">
          <property name="title">
           <string>Groups</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayoutGroups">
           <item>
            <widget class="QCheckBox" name="chkFwd">
             <property name="text">
              <string>fwd</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkBwd">
             <property name="text">
              <string>bwd</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxPeriods">
          <property name="title">
           <string>Periods</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayoutPeriods">
           <item>
            <widget class="QCheckBox" name="chk1">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chk2">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayoutPeriodsCombine">
             <item>
              <widget class="QCheckBox" name="chkCombine">
               <property name="text">
                <string>Combine</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtFirst">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblMinus">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtSecond">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutLabel">
        <item>
         <widget class="QLabel" name="lblSimFitLabel">
          <property name="text">
           <string>Label for simultaneous fit: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtSimFitLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutDataset">
        <item>
         <spacer name="horizontalSpacerDatasetLeft">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnPrevDataset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbDataset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNextDataset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerDatasetRight">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::MWRunFiles</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/MWRunFiles.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
